File: HackerRank/Domains/001 Algorithms/001 Warmup/E - Mini-Max Sum [Array Min Max]/mini-max-sum.cpp
#include <iostream>

using namespace std;

int main() {
    int n = 5;
    long long min = 1000000001, max = 0, sum = 0, a;

    while (n--) {
        cin >> a;

        if (a < min) {
            min = a;
        }
        if (a > max) {
            max = a;
        }
        sum += a;
    }

    cout << sum - max << ' ' << sum - min << endl;

    return 0;
}

File: HackerRank/Domains/001 Algorithms/001 Warmup/E - Mini-Max Sum [Array Min Max]/mini-max-sum.py
def mini_max_sum(arr):
    min_sum = sum(arr) - max(arr)
    max_sum = sum(arr) - min(arr)

    print('{0} {1}'.format(min_sum, max_sum))

if __name__ == '__main__':
    arr = list(map(int, input().rstrip().split()))

    mini_max_sum(arr)